packages/validator-js/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@sequelize/validator.js",
```

Comment:
Will this be released as well? Or does it start with only `@sequelize/core`?

Reply:
The goal is to publish a dummy release, to see if it works